debug/include/shm.inc

```diff
@@ -65,11 +64,20 @@
 #define  SHM_HF_MI_ANTSEL	13 /* bit13: Antenna selection (for testing antenna div.) */
 #define  SHM_HF_MI_BT3COEXT	13 /* bit13: Bluetooth 3-wire coexistence (rev >= 13 only) */
 #define  SHM_HF_MI_BTCANT	14 /* bit14: Bluetooth coexistence (antenna mode) (rev >= 13 only) */
-#define SHM_HF_HI		SHM(0x062) /* Hostflags for ucode options (high) */
 #define  SHM_HF_HI_ANTSELEN	0 /* bit0: Antenna selection enabled (rev >= 13 only) */
 #define  SHM_HF_HI_ANTSELMODE	1 /* bit1: Antenna selection mode (rev >= 13 only) */
 #define  SHM_HF_HI_MLADVW	4 /* bit4: N PHY ML ADV workaround (rev >= 13 only) */
 #define  SHM_HF_HI_PR45960W	11 /* bit11: PR 45960 workaround (rev >= 13 only) */
+
+// New SHM addresses; Source: d11.h 578947 2015-08-13 04:46:06Z
+// http://github.com/tuapuikia/asuswrt-phantom/blob/master/release/src-rt-7.14.114.x/src/include/d11.h
+/* Host flags to turn on ucode options */
+#define	SHM_HOST_FLAGS1		SHM(0x05E)
+#define	SHM_HOST_FLAGS2		SHM(0x060)
+#define	SHM_HOST_FLAGS3		SHM(0x062)
+#define	SHM_HOST_FLAGS4		SHM(0x078)
+#define	SHM_HOST_FLAGS5		SHM(0x0D4)
+
```

Comment:
We already have bit definitions for the values of these fields. See SHM_HF_... definitions above.
Please just rename these to SHM_HOST_FLAGS1-5 and just add 4/5 without bit definitions.